### DISCUSSION TITLE
Issu ingest error

### DISCUSSION THREAD

**Discussion by bangundwir** [mayooear/ai-pdf-chatbot-langchain#14] [Q&A]
![image](https://user-images.githubusercontent.com/17474376/226519372-dac4523d-3e08-4e02-88c2-e885afbf8e93.png)

**Comment by yudidina** [comment]
Hello, I got the same problem, did you solve the problem?

**Reply by bangundwir** [reply]
still problematic too

**Reply by disturbed-mystic1** [reply]
same error

**Comment by mayooear** [comment]
> ![image](https://user-images.githubusercontent.com/17474376/226519372-dac4523d-3e08-4e02-88c2-e885afbf8e93.png)

There could be a number of possible causes:
- openai API key run out of credit or corrupted
- problem with your operating system
- Upgrading versions of LangChain and pinecone
- Failure to set `config` folder environment to match pinecone dashboard
- Failure to set pinecone dashboard vector dimensions to 1536
- incorrect file path
- Incorrect settings in `.env` file

**Comment by xubairp** [comment]
I was facing same issue and i resolved it by running the command: 

**pnpm install dependencies**


and then:

**pnpm run dev**
